src/App.js

```diff
@@ -1,11 +1,12 @@
-import HeaderSection from "./component/HeaderSection/Header";
-
+import HeaderComponent from "./component/Header-section/header.js";
```

Comment:
Have you used the correct names of your folder and file? Isn't it HeaderSection instead of Header-section and Header instead of header?